src/app/scenario/step.component.html

```diff
@@ -225,15 +225,13 @@ <h3 class="modal-title">Are you sure you want to close this scenario?</h3>
   <h3 class="modal-title">Are you sure you want to finish?</h3>
   <div class="modal-body">
     <p *ngIf="shouldKeepVmOnFinish; else no_course">
-      Your VMs will be kept until they expire or the course is marked as
-      "complete".
-      <br />
-      Forcing the finish will immediately de-provision your resources. Any VMs
-      that you are using will be wiped.
+      Your virtual machines will be kept until they expire or the
```

Comment:
Can you remove "virtual" here? Seems small but I would like to move away from hard references to types of compute.

Reply:
Understandable. Don't get too specific. I'll updated it to "resources"

Reply:
Updated!